### ISSUE TITLE
XTTS License After Shutdown

### ISSUE THREAD

**Issue by fakerybakery** [coqui-ai/TTS#3490]
Related to #3488

Hi,
With the recent announcement that Coqui is shutting down, would you consider switching the license to a more permissive one, ie Apache 2.0 or MIT? The technology behind it is incredible and it would be great for the OSS community if the model became open sourced.
Thank you!

**Comment by platform-kit** [comment]
Seconding this

**Comment by FrontierDK** [comment]
I also second this

**Comment by iprovalo** [comment]
+1

**Comment by dario248** [comment]
Seconding this

**Comment by erogol** [comment]
Currently we don't have plans to change the license. It'd also be unfair for people who paid for the license.

**Comment by slavakurilyak** [comment]
License remains MPL-2.0 (Mozilla Public License Version 2.0)?

**Comment by fakerybakery** [comment]
Hi @erogol,
Thank you for your response.
It makes sense that you are trying to keep licensing fair for existing purchasers. I have several questions:

1. Will new customers still be able to purchase a commercial license?
2. Who will own the IP of Coqui (& licensing rights) after the shutdown?
3. Have you considered keeping the CPML license until after all commercial licenses have expired (1 year from now) and after that switching to a more permissive license?

Thank you in advance.

**Comment by erogol** [comment]
1. most probably no
2. noone 
3. I think there will be better models in that timeframe and maybe we train one

**Comment by fakerybakery** [comment]
Hi,

So, essentially, nobody new will be able to use it commercially, however Coqui will not earn any revenue from this limitation?

Also, for point 3, even if there are better models, personally, I still feel that XTTS will remain a valuable and important model for many years to come.

Thank you

**Comment by platform-kit** [comment]
@erogol I'm a paying customer, I bought the year long license just recently after the SaaS shutdown was announced.

As a paying customer I am sorry to say that my worst fear was confirmed with this latest announcement. I thought I was paying to use a software that would have a life for at least a year, not just a few weeks. Ideally I had hoped and expected that you guys would continue for many _more_ years. That seemed to be the sentiment a month ago.

Again, as a paying customer (of both the Coqui Studio SaaS, and then the commercial XTTSv2 license), I am more miffed by the prospect of this becoming abandonware than I would be by the prospect of it becoming MIT licensed OSS.  

Think about it. I've already invested not just money but man-hours into integrating the XTTS model into my product, which is now essentially rendered moot, since this product is going nowhere.  So your decisions have actually created a waste/loss of my both time and money. 

By contrast, if the product goes on to have a life as an MIT licensed model that the community adopts and maintains, then I have not incurred a loss of time or cash. Nor do I have to further spend money and time un-doing/re-doing the work of integrating a different model.

If you truly intend to abandon it from a legal point of view there is no difference _to you_ between changing the license to MIT (or whatever equivalent open/highly permissible license), but there is a huge difference for us, the community, and specifically for those of us who paid for the commercial license. Open sourcing is the least you can do if you are not going to support it going forward.

I am very confident that if you asked a handful of your paying customers, they would ALL feel the same way.

Please consider open-sourcing. Option # 3 mentioned above (open-sourcing after 1 year has passed) might be the most courteous to your paying users, because at least then some kind of competitive advantage is conferred.

Thanks.

**Comment by fakerybakery** [comment]
Also, will paying customers who have already integrated the model into their code have to rewrite it because they cannot renew their license?

**Comment by q5sys** [comment]
> Also, will paying customers who have already integrated the model into their code have to rewrite it because they cannot renew their license?

Effectively... yes.  By not being willing to loosen the license for use after the license expires, Coqui is forcing everyone who built something around their platform to re-architect their pipelines/workflows/etc.  It doesn't seem that they care much, due to the comment of "I think there will be better models in that timeframe...".  It would seems that they expect people to be redeveloping their software anyway. It's unfortunate, but that's the result of businesses relying on software that has commercial restrictions through a license.  It's always a 'Sword of Damocles' that someone else controls. 

Everyone who paid for a license now has a 1 year ticking timebomb to find another solution or be in violation of their license. (Unless Coqui decides to change the license after that 1 year ends)

**Comment by harmony-ai-solutions** [comment]
The question in such cases of course is also, what's going on under the hood which we cannot see? Despite Coqui shutting down, there's probably still some money in the company and also Stakeholder interests in regards to their technology, the model, etc., and therefore they might have their hands tied on the model license, even if they wanted to change it to something more open.

However, from a client perspective, I'd second to what @platform-kit said. It's for sure certain that there will be better AI TTS tools in the future, but switching over to a new tech isn't super trivial, and even more complicated, if you've users who built custom voices based on a specific model. Since it's quite impossible to migrate them 1:1 and maintain their characteristics.

Also, I planned adding Coqiu Studio as a backend option into our software for example, so users who owned a Coqui-Studio subscription could have used your backend as another alternative, if they cannot (or didn't want to) run it locally. Actually I'm kinda glad I didn't start working on that yet. Because that shutdown really came out of nowhere.

Now that Coqui is shutting down, I considered to provide a hosted XTTS API for our users instead, since compute would be availiable for this. (Btw what happened / will happen to existing SaaS users now? Are they basically lost in the void now?)

However, this opens up legal questions with the current license, since despite our project is privately + community-funded, we would of course apply different rate limits to non-supporters using such an API, compared to users in higher tiers, which gives it some sort of commercial character after all.

**Comment by q5sys** [comment]
> Despite Coqui shutting down, there's probably still some money in the company and also Stakeholder interests in regards to their technology, the model, etc., and therefore they might have their hands tied on the model license, even if they wanted to change it to something more open.

If we take what @erogol stated... that's not the case.

> 2\. Who will own the IP of Coqui (& licensing rights) after the shutdown?
> 2\. noone

However if he's being honest and truthful... there'd be no one to pursue someone using the purchased license after the 1 year term ends.


> Actually I'm kinda glad I didn't start working on that yet. Because that shutdown really came out of nowhere.
Agreed, I had some ideas, but they were pretty rough so I wasn't going to spend any money on it right now until I can flesh things out more.
What makes this all more questionable to me though... is that the month prior they were being VERY active on Twitter to promote 1 year licenses.  We don't know how many sold, it may have only been a few, but if my business bought a 1 year commercial license from another company, and then 30 days later they closed up shop and stopped providing services... I'd be speaking with my attorney.  Having not bought a license, I don't know what the fine print on the terms were, it's possible the Dec 2023 license sale included some very fine print that was put in so they could just take everyone's money and run. (I hope that's not the case)

I wish the people at Coqui all the best in the future, and I'm sad they had to close up shop.  But promoting people to buy a license and then rug pulling and then refusing to at least allow others to maintain the code for their own license that they just bought... just feels really wrong to me.

**Comment by fakerybakery** [comment]
cc: @JRMeyer

**Comment by pablodz** [comment]
To say only that many companies use really old open-source models like this one, over 4 years old and still in production. The fact that there are better and more efficient models doesn't mean that the community doesn't use the old ones, as mentioned above. It's a significant investment to switch from one framework to another, and companies prefer to bear the extra cost for a longer period until they are compelled to use the newer ones.


![image](https://github.com/coqui-ai/TTS/assets/28235457/cef1f8b3-c55f-4305-bf22-d653f96b9153)

**Comment by FrontierDK** [comment]
This is true. I have helped a lot of companies, and to save cost, a lot of companies run outdated software - usually in virtual enviroments, where broken hardware isn't an issue. I still have a few clients who run win95/nt4 to control production equipment. So Coqui can, without problems, run in a virtual enviroment for decades.

**Comment by SomeDevWeb** [comment]
YES, PLEASE make it open-source!!! 🥺 Everybody would benefit, including Coqui...

OK, I have a clarifying question then. If I install the models on my personal PC and generate some voices that way, am I allowed to use them for a youtube video? Thanks!

**Comment by fakerybakery** [comment]
@SomeDevWeb

> OK, I have a clarifying question then. If I install the models on my personal PC and generate some voices that way, am I allowed to use them for a youtube video? Thanks!

IANAL, but I'm pretty sure the license prohibits commercial use of any audio outputted from the model, even if it's on your own computer, unless you own a license. It would be super sad if people can't use this model any more, please consider open sourcing it @erogol @JRMeyer.

According to Coqui AI's [FAQs](https://coqui.ai/faq),

> **Q. Can I sell content that I produced after the license finishes?**
> Yes. If you make an audiobook today with XTTS, you can sell that audiobook even after your XTTS license expires.
>
> **Q. Is this licensing for unlimited web-based SaaS API usage or is the license for self-hosting / on-premise / locally running the model?**
> It's a bring-your-own GPU deal. You can run XTTS on your servers, on a macbook, on a desktop computer... wherever you want. While you have an active license, they're your model weights, you run them where you want.

DISCLAIMER: I am not a lawyer. The above text is not legal advice.

**Comment by SomeDevWeb** [comment]
> IANAL, but I'm pretty sure the license prohibits commercial use of _any_ audio outputted from the model, even if it's on your own computer, unless you own a license. 

Hi. Sorry, I kinda disagree, after what I saw here:

**Q. Do I need the license if my use-case isn't commercial?**
Nope. If you're using XTTS non-commercially, it's free. You're already covered under the default license (the [Coqui Public Model License](https://coqui.ai/cpml)).

And in that licence it says:
**Non-commercial Purpose**
Non-commercial purposes include any of the following uses of the model or its output, but only so far as you do not receive any direct or indirect payment arising from the use of the model or its output.
**_Personal use for research, experiment, and testing for the benefit of public knowledge, personal study, private entertainment, hobby projects, amateur pursuits, or religious observance._**

... which is exactly what one would do with a video.
Of course, there's the consideration of the potential youtube monetization, which presumes the video is not producing any income. Only in the future if potentially the channel would become profitable, then in theory I suppose there will be a need to buy a licence, which obviously can't be done, so then I presume that's that...

**But still, it's SUCH A BIG SHAME for this project to die. I really hope someone will continue it.** 
In reality, I think all it needs is a fork of the FOSS repo and someone (or more people) to lead...

But who knows, we'll see what happens...

**Comment by q5sys** [comment]
> In reality, I think all it needs is a fork of the FOSS repo and someone (or more people) to lead...

It's not the repo that's the issue. This repo and everything in it is under the [MPL-2.0 license](https://github.com/coqui-ai/TTS#MPL-2.0-1-ov-file)
The issue is that the models which are on Huggingface are under the [CPML](https://coqui.ai/cpml).
Anyone/Everyone can fork this repo and keep using XTTS under the MPL 2.0 license.  But someone would have to pitch in and re-create the Models that Coqui made.
We cant just retrain the model, because that's still using their original model and could be considered an output from that model. We need a custom freshly trained model with open weights.  

Coqui could re-license the model, but they have indicated that they wont.
Coqui could publicly release details of their initial training pipeline and information about their datasets, but I doubt they will. 
 
They have stated that "no one" will own the IP or rights.  So what value is there in keeping it private?  Personal benefit.  The knowledge of what they did and how has value... My guess is that the employees are hoping that their private knowledge of how it works will land them a job somewhere, so they are incentivized -not- to share anything that they don't have to. 
If they truly believed there's no value to any of it... then there would be no reason to let it be lost.

The shutdown coming so soon after they went on [a social media campaign in Dec of 2023 selling 1 year licenses](https://archive.is/9T44Z) really rubs me the wrong way, and I cant help but think there's something else at play.  Maybe there's not, maybe they really did try to get people to buy something they knew they would kill off one month later.  But with silence about the reasons and not being willing to open source the models, I cant help but assume that there's details that are private that explain why everything has gone the way it has.

**Comment by metal3d** [comment]
> Currently we don't have plans to change the license. It'd also be unfair for people who paid for the license.

It's unfair to leave some people having a "for life" right to freely use a model, and to not authorize new users to have the same rights.

XTTS model is fantastic, and I'm sad that the society couldn't continue. Really. But, now... it's too late.

Think about the Blender foundation. Blender was proprietary but "freely usable" until NaN company shut down. Ton Roosendaal made a hard work to make Blender open source and to free to use. Just take a look on it now...

Restriction is not good at all, it only makes the project to be unusable in the real world (the world where we need money, yep). 

Leaving the XTTS model freely usable could help donators to give a coin.

**Comment by fakerybakery** [comment]
+1 @metal3d 

> It's unfair to leave some people having a "for life" right to freely use a model, and to not authorize new users to have the same rights.

I don’t think anyone has a license for life, the licenses were all 1 year, right? Unless there were licenses issued before that

**Comment by metal3d** [comment]
> +1 @metal3d
> 
> > It's unfair to leave some people having a "for life" right to freely use a model, and to not authorize new users to have the same rights.
> 
> I don’t think anyone has a license for life, the licenses were all 1 year, right? Unless there were licenses issued before that

Yes that's right.

But what happens after one year for those who are based their business on this model?

And there is a weird mention about the model "in the form provided by the licensor". What is this form? Is a fine tuned model not in the form provided by the licensor, and so, not covered by the CMPL?

The license seems nebulous and lacking in detail. It only provides restrictions and not authorizations in the case of fine tuning, and above all it does not indicate the conditions in the case of purchase (which is no longer possible) of commercial exploitation rights.

I'm not a lawyer, but it seems to me that there are gaps and that I could hardly be asked for trouble if I train my model from the model supplied, even in the case of fine tuning.

Of course, the xtts owner will say that I don't have the right to use it for my futur monetized YouTube video... It's a pity. Xtts is impressive, I fine tuned the model with my dataset and the generated voice is near the perfection. My wife didn't find the difference between my voice and the generated one.

**Comment by fakerybakery** [comment]
I think the only option left is to re-train XTTS since the code is MPL licensed, but that would be very expensive

**Comment by harmony-ai-solutions** [comment]
> I think the only option left is to re-train XTTS since the code is MPL licensed, but that would be very expensive

We might be able to give that a try when we finished training our new one-shot TTS; but we'll still be busy with that for some time.

However I didn't check the license in detail; is it possible to use custom trained weights trained from scratch for this model in commercial context, too ?

**Comment by metal3d** [comment]
That my question about the "form provided by the licensor" in the CMPL license. A fine tuned model isn't the model in the same form, and it's not an output of the model. It's a derived model.

The CMPL is absolutely unclear on this point, and doesn't even mention it. If I were a little dishonest, I'd say that my model is not the one provided by CoquiTTS, and that I therefore have no restrictions on using it as I see fit.

In fact, I'm sure this goes against their desire to restrict commercial use of the XTTS V2 model. And that makes me a little bitter. I've always found this kind of restriction terribly harmful. The only way for CoquiTTS to be revived commercially is for people to use it and ask for support.

In my opinion, this is where CoquiTTS went wrong. They wanted to sell access to the studio **and** sell the model... Many companies have tried this model and failed.

I'll say it again, but that's what happened with Blender, and it was a disaster for the publisher. What saved Blender was precisely that it gave everything to the community. The money came from donations and the sale of support, but not from the product.

Hybrid open source is almost always doomed to failure. You either have to go full-opensource, or full-ownership (and have some pretty strong arguments in favor of this mode of operation).

So, to sum up: a fine-tuned model is not mentioned in the license, we don't have an answer on the subject, and it seems imperative that the CoquiTTS managers rid us of these absurd clauses since the box has crashed.

**Comment by fakerybakery** [comment]
> > I think the only option left is to re-train XTTS since the code is MPL licensed, but that would be very expensive
> 
> We might be able to give that a try when we finished training our new one-shot TTS; but we'll still be busy with that for some time.
> 
> However I didn't check the license in detail; is it possible to use custom trained weights trained from scratch for this model in commercial context, too ?

The code itself is MPL licensed, only the weights are licensed under CPML. So if you completely retrained everything from scratch you probably could (but disclaimer IANAL), finetunes would probably still fall under the license. But I might be wrong

**Comment by louispaulet** [comment]
IANAL, but retraining a model for legal reasons is ecologically wrong.  

If you do not change the license for future hobbyists, at least do it for the planet.

**Comment by choilive** [comment]
> IANAL, but retraining a model for legal reasons is ecologically wrong.
> 
> If you do not change the license for future hobbyists, at least do it for the planet.

Eh, ecologically wrong? How so? Training cost? Training a model costs basically nothing in terms of energy.  The internet uses about 2% of global energy, maybe we should shut that down too.

**Comment by Dmmoim** [comment]
We won't see them change the license since it is now part of Rabbit r1. Its co-founder works there. :(

**Comment by platform-kit** [comment]
Lame. They've been outclassed by other models anyway, but still, this is lame.

**Comment by jneuendorf** [comment]
> Lame. They've been outclassed by other models anyway, but still, this is lame.

Do you have any reference to better (open-source) alternatives?

**Comment by platform-kit** [comment]
@jneuendorf https://github.com/metavoiceio/metavoice-src

1.2B Parameters vs XTTS's 750m.

**Comment by pablodz** [comment]
Forked :D https://github.com/idiap/coqui-ai-TTS

**Comment by eginhard** [comment]
> Forked :D https://github.com/idiap/coqui-ai-TTS

Just to clarify that this fork doesn't change the license of the code (MPL 2.0) or the license of any pretrained models, including XTTS (CPML), so it's not really relevant to this issue.

**Comment by lukaLLM** [comment]
Any updates regarding it or what alternative do you use this looked like the best low-latency commercial solution. I go into dependency searching for tool hell to replace it

**Comment by danders341** [comment]
hey there! Anyone knows where else I can buy XTTS license? Their payment link is not working. Thanks in advance

**Comment by danders341** [comment]
@erogol 
hey there! Anyone knows where else I can buy XTTS license? Their payment link is not working. Thanks in advance

**Comment by fcnjd** [comment]
@danders341 I'm sorry, but there's no way you could renew your license, Coqui has shutdown and, as written above in this discussion, no one owns the right anymore.

**Comment by fcnjd** [comment]
@erogol How is the current state of this? Now that one year has passed, have you re-considered making it open now? The aspect of unfairness now isn't a reason anymore, as @platform-kit pointed out very well. Also, you assumed that now XTTS would be superseeded by newer models - I don't think so. Look, on Huggingface in the TTS category, it's STILL on position 2 in the ranking. Also, I couldn't find any other model, supporting that many language and being this far fine-tuned. Long story short, it could still be of much value to open this model.
Source: [Huggingface TTS models](https://huggingface.co/models?pipeline_tag=text-to-speech&sort=trending)

**Comment by rsxdalv** [comment]
> @erogol How is the current state of this? Now that one year has passed, have you re-considered making it open now? The aspect of unfairness now isn't a reason anymore, as @platform-kit pointed out very well. Also, you assumed that now XTTS would be superseeded by newer models - I don't think so. Look, on Huggingface in the TTS category, it's STILL on position 2 in the ranking. Also, I couldn't find any other model, supporting that many language and being this far fine-tuned. Long story short, it could still be of much value to open this model. Source: [Huggingface TTS models](https://huggingface.co/models?pipeline_tag=text-to-speech&sort=trending)

Coqui has a very good ranking and SEO, maybe because it was actually open source for a long time before they decided to make their own model based on tortoise tts and not open source it. Then Google's money into Mozilla stopped and the rest is history. Yes the model is still very much in the "good quality, predictable results" category, but there are plenty of new models too - that can either generate faster, with less resources or with more options.

**Comment by danders341** [comment]
@erogol so what happens with all the existing services using your technology? Because I could give it a try on replicate and it’s still working. As you said no one owns the rights anymore but I can still run it on different services online.

**Comment by q5sys** [comment]
> As you said no one owns the rights anymore

This is  incorrect.  That's not how copyright works. The person who created the work retains the copyright even if they aren't doing anything with it.  In order for the creator/owner to relinquish their rights is to legally declare that it is in the public domain or that they have transferred copyright to another entity.   

The copyright was held by the corp behind Coqui.  They are the rights holder to the software.  We don't know what legally happened with that entity, did it properly close and all the paperwork get filed with the state/fed goverment?  If so then the rights would fall back to the person who wrote the code.  

Them saying "No one owns this" is not legally actionable.  I spoke with an attorney about this last year when all this first went down and I and a few others were trying to see if there was anyway this could be salvaged and open sourced.

We need a clear statement from either @erogol or @JRMeyer about whether they are legally relenquishing the copyright of the code/models to public domain, or if they are changing the code/model to be under an MIT/BSD/GPL license.

I really hope they do, Coqui-TTS is still IMHO the best TTS out there.  And it's very unfortunate that its' just been left to bitrot and be held captive by the license its under.  I can try to reach out to @JRMeyer and see if we can get some clarity.

**Comment by danders341** [comment]
@q5sys @rsxdalv @erogol @JRMeyer Im sorry for the confusion but I’m not asking them to make it open. I’ll happily pay for using their hard work. I’m just looking for a place where I can buy the license. 
Once again, I’m not here asking them to make their research and work fully public but instead I’m looking to buy pay for the license. Can you @erogol @JRMeyer please react? The next place I’ll try to reach you guys is Cantina. Thanks in advance for your time :)

**Comment by q5sys** [comment]
@danders341 
I understand you're willing to pay for a license, but if the company doesn't exist anymore... then it will be impossible for you or anyone else to purchase a license.  They're not going to go through the time cost and expense to restart a company just to sell a few licenses.
Since we (the community) don't know what the legal status is of Coqui there's nothing any of us can do until those at Coqui make a statement. 
It's possible the company is still dormant and they can just turn on the payments page, but I doubt that's the case, since @erogol stated that no one will own the IP, which leads me to believe the company doesn't exist anymore.

**Comment by q5sys** [comment]
Hmm... It looks like Coqui does still have an active registration.
![image_2025-01-04_12-11-39](https://github.com/user-attachments/assets/150527ce-5c98-426f-b80d-f2b961f48866)
(unless there's another Coqui that's in Berlin)
![image_2025-01-04_12-12-56](https://github.com/user-attachments/assets/8264e60d-8312-4e61-bc5e-dcbc17c2d864)

**Comment by Totaie** [comment]
I’ve been exploring the XTTS model and noticed that Coqui has seemingly shut down. Given this, I’m curious about the status of the Coqui Public Model License (CPML). Is the license still enforceable even though the company is no longer operational? It feels somewhat unfair to developers that we’re left in a grey area, especially those of us who are looking to develop commercial projects.

Additionally, are there any alternative models you recommend that Offer super-fast performance, can be run locally, provide great quality voice cloning for short segments, and support commercial use?
Any guidance or suggestions would be greatly appreciated.

**Comment by platform-kit** [comment]
@Totaie CosyVoice2 is apache licensed and superior to Coqui in terms of features, performance, AND quality.

**Comment by q5sys** [comment]
Personally I find CosyVoice2 to be way more robotic and obviously artificial.  The quality of the voice from xtts2 is far better IMHO.  That's why I continue to use it.

**Comment by platform-kit** [comment]
@q5sys I've done extensive testing. It is weaker with accents that are out of scope, but you could always fine tune to add those. However with in-scope voices, its prosody is not only better, it's controllable. Not to mention it has twice as many parameters as XTTS. Look at the paper.

For commercial usage, there is no reason to use XTTS at this point. CosyVoice is the clear leader, it even outperforms VoiceCraft.

**Comment by fcnjd** [comment]
But it supports way less languages then XTTS. Western/Southern European languages, like French/Spanish/German aren't supported at all by CosyVoice, but by XTTS.

**Comment by platform-kit** [comment]
@fcnjd yes, but you can train your own and do whatever else you want with the model and its outputs, including build commercial applications, due to the Apache license. 

XTTS is dead for commercial purposes.

**Comment by q5sys** [comment]
> Look at the paper.

I did look at the paper, and I tested it myself.  Maybe it'll get better in time, but for what I do and what I want XTTS is better.  All of the emotional expressions that Cosy does sound fake and like they're from a person who doesn't actually know how that emotion should sound.  "it can do more" is of little concern if the voice itself doesn't have the right timbre, meter, intonation, etc. 
I dont want more options that sound worse.  I dont like Cosy, so I'm not going to use it.  If you like Cosy, good for you. Go and use it and leave us alone.  

This isn't the Cosy repo. This isn't the Cosy issue tracker.  This is the Coqui repo where we are discussing and requesting status updates on the project.

Dont shill for another project here.

**Comment by platform-kit** [comment]
@q5sys I'm not shilling, I am an entrepreneur who was burnt by coqui's shutdown. 

Your attitude is out of whack, don't treat github like a videogame forum, this is not the place to flame people. Grow up.

**Comment by q5sys** [comment]
@platform-kit You've gone into one developers software repo and are trying to convince people to use different software from other developers repo. 

That is pretty much unacceptable behavior in any open source community.

You wouldn't go into the Fedora Repos and tell people to use Ubuntu.
You wouldn't go into the KDE Repos and tell people to use Gnome.
For the same reasons...
You shouldn't go into one TTS Repo and tell people to use another TTS.

You aren't the only one who has gotten burned by the Coqui shutdown, many people have.  But that doesn't mean people can behave inappropriately.  That's why I called it out. 

This is github, not reddit.

**Comment by platform-kit** [comment]
@q5sys this thread is about the shutdown of this project, it is only natural to suggest alternatives, which is exactly what @erogol (creator of Coqui) did himself.  

You are being immature.

**Comment by q5sys** [comment]
> it is only natural to suggest alternatives, which is exactly what @erogol (creator of Coqui) did himself.

No he did not do that.  Please do not lie.  People can scroll up and see exactly what he said.  He said... 

> 3\. I think there will be better models in that timeframe and maybe we train one

That is not a direct recommendation of another project.   He did not say "Go use this other project".  Erogol did not suggest/name another project he should use. 

In multiple comments you have told people to go use CosyVoice.   So, no, you are not doing the "exact" same thing that he did.  

I don't even care about the initial suggestion that you made in response to Totaie.  But when two different people mentioned that we think Coqui is better; you decided to argue with us.  Your suggestion was one thing, turning it into an argument about why we were wrong for liking Coqui, is entirely different.   

You're arguing in Coqui's Official repo about why we are wrong for liking Coqui's software.  
I don't think I'm the one being immature and disrespectful. And on that note...
I'm not the one editing comments to add insults...  you are.
![image](https://github.com/user-attachments/assets/2e5981cc-cee8-4ffa-9739-c50f81c16368)

**Comment by rsxdalv** [comment]
> That is pretty much unacceptable behavior in any open source community.

CPML licensed community. Perhaps the current situation is the exact result of decisions made beforehand, and arguing isn't entirely productive, honestly it's like a nail in the coffin.

Instead, since customer is always right in matters of taste, here are just some random models that are all "allegedly" alternatives:
Tortoise (finetuned + optimized),
Parler TTS (more control, limited quality)
MARS5 (commercial use might be an issue?)
Vall-e-x
StyleTTS2
F5-TTS (current flavour of the quarter)
... and many others ...

Additionally, there are now dedicated text to singing models.

**Comment by marktellez** [comment]
For those who got screwed here, give me a message. I have rearchitected and retrained an equivalent model to xtts - id be happy to hook you up. demo is at https://voxbird.ai - it uses gpt, latents, and an rvc like gan

**Comment by fakerybakery** [comment]
> For those who got screwed here, give me a message. I have rearchitected and retrained an equivalent model to xtts - id be happy to hook you up. demo is at https://voxbird.ai - it uses gpt, latents, and an rvc like gan

Is it open sourced?

**Comment by marktellez** [comment]
Nope. But i allow comercial use, and training/finetuning

*MarkTellez*
*Sr. Software Engineer & AI Researcher*
Get live help from an expert programmer at https://devmentor.live
